ci-operator/step-registry/openshift-qe/run-api-apf-customized-flowcontrol/openshift-qe-run-api-apf-customized-flowcontrol-ref.yaml

```diff
@@ -1,9 +1,6 @@
 ref:
   as: openshift-qe-run-api-apf-customized-flowcontrol
-  from_image:
-    namespace: ci
-    name: ocp-qe-perfscale-ci
-    tag: latest
+  from: ocp-qe-perfscale-ci
```

Comment:
Did we skip `cli: latest` intentionally here?

Reply:
not intentionally, will update!